bin/update-package.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+git_setup()
+{
+  cat <<- EOF > "$HOME/.netrc"
+		machine github.com
+			login $GITHUB_ACTOR
+			password $API_TOKEN_GITHUB
+		machine api.github.com
+			login $GITHUB_ACTOR
+			password $API_TOKEN_GITHUB
+EOF
+  chmod 600 "$HOME/.netrc"
+
+  git config --global user.email "$GITHUB_ACTOR@users.noreply.github.com"
+  git config --global user.name "$GITHUB_ACTOR"
+}
+
+# Halt on error
+set -e
+
+git_setup
+
+BASE=$(pwd)
+COMMIT_MESSAGE=$(git show -s --format=%B $GITHUB_SHA)
+
+git config --global user.email "$GITHUB_ACTOR@users.noreply.github.com"
+git config --global user.name "$GITHUB_ACTOR"
+
+echo "Cloning folders in /packages and pushing to Automattic package repos"
+
+# sync to read-only clones
+for package in packages/*; do
+	[ -d "$package" ] || continue # We are only interested in directories (i.e. packages)
+
+	cd $BASE
+
+	# Only keep the package's name
+	NAME=${package##*/}
+
+	echo " Name: $NAME"
+
+	CLONE_DIR="__${NAME}__clone__"
+	echo "  Clone dir: $CLONE_DIR"
+
+	# Check if a remote exists for that package.
+	$( git ls-remote --exit-code -h "https://github.com/automattic/jetpack-${NAME}.git" >/dev/null 2>&1 ) || continue
+	echo "  ${NAME} exists. Let's clone it."
+
+	# clone, delete files in the clone, and copy (new) files over
+	# this handles file deletions, additions, and changes seamlessly
+	git clone --depth 1 git@github.com:automattic/jetpack-$NAME.git $CLONE_DIR &> /dev/null
```

Comment:
This is currently not working. I'm having trouble authenticating within the action to clone, or even commit to any repo.